tests/nosetests/pyanaconda_tests/product_test.py

```diff
@@ -217,6 +217,11 @@ def fedora_products_test(self):
              "fedora-silverblue.conf"],
             WORKSTATION_PARTITIONING
         )
+        self._check_default_product(
+            "Fedora", "IoT",
```

Comment:
Could you please also change the product name here? It should fix the failing test.

Reply:
Done.